src/pyorderly/outpack/filestore.py

```diff
@@ -79,6 +79,11 @@ def onerror(func, path, _exc_info):
             if not os.access(path, os.W_OK):
                 os.chmod(path, stat.S_IWUSR)
                 func(path)
+            else:
+                # This is always called by rmtree from within a try-catch
+                # block, but the linter doesn't see that and complains about
+                # the argument-less `raise` statement.
+                raise  # noqa: PLE0704
```

Comment:
I'd removed this line in #57 to appease the linter, but turns out the line is actually useful and doing so breaks the tests on Windows.